dev-system/dev/user.clj

```diff
@@ -117,6 +117,20 @@
   []
   (set-modes! :all settings/default-run-mode))
 
+(defn- distribute-schemas
+  "A function that will copy the directory OtherSchemas in to all the necessary
+   apps and will be called on reset"
+  []
+  (let [path (System/getProperty "user.dir")
+        apps ["search-app/resources"
+              "ingest-app/resources"
+              "indexer-app/resources"
+              "metadata-db-app/resources"
+              "system-int-test/resources"]
+        source "schemas"]
+    (doseq [app apps]
+      (shell/sh "cp" "-r" (clojure.string/replace path #"dev-system" source) (clojure.string/replace path #"dev-system" app)))))
```

Comment:
the destination path must always be lower case. CMR will use the lowercase value for the generic name (grid vs Grid). In this way we don't have to worry about how the schema was input in the repository and CMR does not have to do a harder search on the file system.